shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/asserts/statement/dal/impl/SetVariableStatementAssert.java

```diff
@@ -42,11 +42,12 @@
      * @param expected expected variable statement test case
      */
     public static void assertIs(final SQLCaseAssertContext assertContext, final SetStatement actual, final SetVariableStatementTestCase expected) {
-        if (null != expected.getVariable()) {
-            assertNotNull(assertContext.getText("Actual variable expression should exist."), actual.getVariable());
-            assertThat(assertContext.getText("variable expression assertion error: "), actual.getVariable().getVariable(), is(expected.getVariable()));
+        if (null != expected.getVariableName()) {
+            assertNotNull(assertContext.getText("Actual variable expression should exist."), actual.getVariableExprList());
+            assertThat(assertContext.getText("variable expression assertion error: "), actual.getVariableExprList().get(0).getVariable().getVariableName(), is(expected.getVariableName()));
```

Comment:
Why judge the first variable here? 
FIY: `actual.getVariableExprList().get(0)`

Reply:
yes,i know ，but this time code have commit too much file，I want to optimize multivariable test on next time....there code will can check  a single variable  and a single variable value..